.travis.yml

```diff
@@ -17,21 +17,22 @@ matrix:
       env:
         - CS_CHECK=true
     - php: 7.1
-    - php: hhvm 
+    - php: hhvm
   allow_failures:
     - php: hhvm
 
 before_install:
   - if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
   - composer self-update
   - if [[ $TEST_COVERAGE == 'true' ]]; then composer require --dev --no-update satooshi/php-coveralls:^1.0 ; fi
+  - if [[ $TEST_COVERAGE == 'true' ]]; then composer require --dev --no-update michaelmoussa/php-coverage-checker:^1.1.0 ; fi
```

Comment:
Ah, okay, that makes sense! Coveralls fails the build if the change indicates a _decrease_ in coverage, per a threshold you set on the service (defaults to 0.01%, IIRC, which is often unreasonable, as simple syntax changes can sometimes result in trivial coverage changes). I like the idea of setting a _minimum acceptable coverage percentage_, however; that feels like a better metric.

Thanks!